views/layouts/_generic.njk

```diff
@@ -1,6 +1,7 @@
 {% extends "template.njk" %}
 
 {% set htmlClasses = 'no-js' %}
+{% set assetUrl = 'https://design-system.service.gov.uk/assets' %}
```

Comment:
Non-blocking but I wonder if there's a metalsmith variable of config that we could use here.